src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { FullTodo } from '../../types/FullTodo';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+type Props = {
+  todo: FullTodo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    user,
+    id,
+    completed,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={
+        classNames('TodoInfo', { 'TodoInfo--completed': completed })
```

Comment:
Consider using a more descriptive class name for better readability and understanding. 🤔